Makefile

```
@@ -63,3 +81,6 @@ format: $(GO_BIN)/gofumpt $(GO_BIN)/gci
 	@echo "==> Formatting project files"
 	gofumpt -w .
 	gci write -s standard -s default .
+
+mocks: $(MOCK_DIR)/*.go
+	@echo "==> Mocks generated"
```

```suggestion
	@echo "==> Mocks generated"

```
Just missing a new line at the end.